src/MainWindow.ui

```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>946</width>
-    <height>632</height>
+    <height>1081</height>
```

Comment:
i think that's a problem.... @deXol ?

Reply:
I don't remember explicitly changing these sizes, must be qt creator's doing. Will double-check tomorrow.

Reply:
Fixed